code/src/App.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react'
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Detail from './pages/Detail'
+import List from './pages/List'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Routes>
+      <Route exact path="/" element={<List/>}>
```

Comment:
Why is this sent as an element? 

We did it like this: `<Route exact path="/" component={MainSite}>`

We are curious how it works!